redisdb/test_redisdb.py

```diff
@@ -0,0 +1,363 @@
+# stdlib
+import logging
+import pprint
+import random
+import time
+
+# 3p
+from distutils.version import StrictVersion # pylint: disable=E0611,E0401
+from nose.plugins.attrib import attr
+from nose.plugins.skip import SkipTest
+import redis
+
+# project
+from checks import AgentCheck
+from tests.checks.common import AgentCheckTest, load_check
+
+logger = logging.getLogger()
+
+MAX_WAIT = 20
+NOAUTH_PORT = 16379
+AUTH_PORT = 26379
+SLAVE_HEALTHY_PORT = 36379
+SLAVE_UNHEALTHY_PORT = 46379
+DEFAULT_PORT = 6379
+MISSING_KEY_TOLERANCE = 0.6
+
+
+@attr(requires='redisdb')
+class TestRedis(AgentCheckTest):
+    CHECK_NAME = "redisdb"
+
+    def test_redis_auth(self):
+        # correct password
+        r = load_check('redisdb', {}, {})
```

Comment:
Ignore at will: I normally prefer to `self.load_check({}, {})` and then use `self.check`... Mainly because it's what the templated check does. I think it should do the trick and it'd be one less import :)